### DISCUSSION TITLE
Trouble using Plot.geo

### DISCUSSION THREAD

**Discussion by nacho-cs** [observablehq/plot#2056] [Q&A] [answered]
I have tried using the Plot.geo mark, however I cannot get it to work. I have tried with both these datasets: https://www.data.gouv.fr/fr/datasets/contours-des-communes-de-france-simplifie-avec-regions-et-departement-doutre-mer-rapproches/ and https://d3js.org/us-10m.v1.json. 

Here is what my code looks like:

```
import "./style.css";
import * as Plot from "@observablehq/plot";
import * as topojson from "topojson-client";
import * as d3 from "d3-fetch";

const div = document.querySelector("div");

async function main() {
  const usTopojson = await d3.json("https://d3js.org/us-10m.v1.json");
  let { features } = topojson.feature(
    usTopojson,
    usTopojson.objects.counties
  );

  const chart = Plot.plot({
    projection: "albers-usa",
    marks: [Plot.geo(features, {stroke: "black"})],
  });
  div.append(chart);
}

main();
```

This is suprising to be because this example is basically copy pasted off the docs.

**Comment by mbostock** [comment, marked as answer]
The dataset you’re using is already projected, so you can’t reproject it with _albers-usa_. You have to use the null projection instead, or switch to a dataset that’s unprojected and then you can use whatever projection you want.

**Reply by nacho-cs** [reply]
Thank you!
